vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/CallItemFactory.kt

```diff
@@ -101,7 +101,7 @@ class CallItemFactory @Inject constructor(
                 createCallTileTimelineItem(
                         roomSummary = roomSummary,
                         callId = callEventGrouper.callId,
-                        callStatus = if (callEventGrouper.callWasMissed()) CallTileTimelineItem.CallStatus.MISSED else CallTileTimelineItem.CallStatus.ENDED,
+                        callStatus = if (!callEventGrouper.callWasAnswered()) CallTileTimelineItem.CallStatus.MISSED else CallTileTimelineItem.CallStatus.ENDED,
```

Comment:
Can I ask you to remove `!` and swap if and else blocks to simplify reading?

Reply:
was about to say the same thing :smile:

Reply:
:). also the CI complains that the line is too long

Reply:
Done